code/game/turfs/space/space.dm

```diff
@@ -156,7 +156,7 @@
 		var/atom/movable/AM = A.pulling
 		A.forceMove(DT)
 		if(AM)
-			var/turf/T = get_step(A.loc,turn(A.dir, 180))
+			var/turf/T = A.loc
```

Comment:
This is changing it so that the pulled object is just moved directly on to the tile of the thing pulling it instead of staying behind it through the transition. You should deal with the actual problem instead of this workaround.